Examples/Bonds/Bonds.cpp

```diff
@@ -41,8 +41,8 @@
 #include <ql/time/daycounters/actual360.hpp>
 #include <ql/time/daycounters/thirty360.hpp>
 
-#include <iostream>
-#include <iomanip>
+
+#include<bits/stdc++.h>
```

Comment:
This is a gcc-only header and won't work with Visual Studio and clang.

Reply:
ok thank u